h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -1978,39 +1987,23 @@ private void scoreAndUpdateModel(){
       long t1 = System.currentTimeMillis();
       Frame train = DKV.<Frame>getGet(_parms._train); // need to keep this frame to get scoring metrics back
       _model.score(train).delete();
-      scoreEnd(train, t1);
-/*
```

Comment:
remove commented out code

Reply:
unresolving this comment since the code was not removed

Reply:
It is done.

Reply:
It is removed as I am checking now.